### PR TITLE
Enable keyword arguments when requesting metafields

### LINKED ISSUE #135: metafields() method should be able to take options parameters for limit and page
```
import shopify
product = shopify.Product.find(5972485446)
metafields = product.metafields()
print(len(metafields))
> 50
metafields = product.metafields(limit=250)
> TypeError: metafields() got an unexpected keyword argument 'limit'
```
I looked into the code, and it seems like it may be a simple fix. If I come up with something I'll submit a pull request, if I can't I'll let it be known here so somebody else can try tackling it.

### PR DESCRIPTION
**Pull request by flux627**
Resolves #135

### CODE REVIEW

**Comment by kevinhughes27** [comment]
Can you add some tests please!

**Comment by flux627** [comment]
I added tests for the `metafields_count` method, even though it looks like no other resource tests for count functionality. I can remove it if you think it's not necessary.

File: test/fixtures/metafields_count.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"count":2}
```
Comment (kevinhughes27) [inline test/fixtures/metafields_count.json]: can you add the newline char here please

File: shopify/mixins.py [hunk 2]
```diff
@@ -11,8 +11,15 @@ def count(cls, _options=None, **kwargs):
...
-    def metafields(self):
-        return shopify.resources.Metafield.find(resource=self.__class__.plural, resource_id=self.id)
+    def metafields(self, _options=None, **kwargs):
+        if _options is None:
+            _options = kwargs
+        return shopify.resources.Metafield.find(resource=self.__class__.plural, resource_id=self.id, **_options)
+
+    def metafields_count(self, _options=None, **kwargs):
+        if _options is None:
+            _options = kwargs
```
Comment (kevinhughes27) [inline shopify/mixins.py]: can you explain this `_options` code please

Comment (flux627) [inline shopify/mixins.py]: I took the pattern from the `count` method of the `Countable` class, seen just above in the same file. Basically, it allows you to either specify parameters with a dictionary or with keyword arguments. This is consistent with the behavior of the rest of the API.

Comment (kevinhughes27) [inline shopify/mixins.py]: okay cool that makes sense! Sorry I am just not super familiar with python or this repo anymore so figured I would ask.

**Comment by flux627** [comment]
Anything else needed before we're good to merge?

**Comment by kevinhughes27** [comment]
nope. Thanks for the PR!

**Pull request merged**